src/main/java/org/broadinstitute/hellbender/utils/recalibration/BaseRecalibrationEngine.java

```diff
@@ -244,7 +244,8 @@ private void updateRecalTablesForRead( final ReadRecalibrationInfo recalInfo ) {
         final int readLength = read.getLength();
         for( int offset = 0; offset < readLength; offset++ ) {
             if( ! recalInfo.skip(offset) ) {
-                for (final EventType eventType : EventType.values()) {
+                for (int idx = 0, n = EventType.valuesCount(); idx < n; idx++) { //Note: we loop explicitly over cached values for speed (see EvenType doc).
```

Comment:
This has the sad side effect of looking more complicated than it is.  Do you think this is better than just caching `EventType` as a private static array  in BaseRecalibrationEngine and using that?